src/parser/core/EventLinkNormalizer.ts

```diff
@@ -35,15 +35,15 @@ export type EventLink = {
   /** The maximum allowed timestamp difference *backward in time from the linking event to the
    * referenced event* in order for a link to be made. Defaults to 0 ms when omitted. */
   backwardBufferMs?: number;
-  /** Iff true, the two events may be linked even with different sources.
+  /** If true, the two events may be linked even with different sources.
    * In most cases this should be false, and will default to false when omitted */
   anySource?: boolean;
-  /** Iff true, the two events may be linked even with different targets.
+  /** If true, the two events may be linked even with different targets.
    * In most cases this should be false, and will default to false when omitted */
   anyTarget?: boolean;
-  /** Iff defined, links will also be added from the referenced event to the linking event using this relation. */
+  /** If defined, links will also be added from the referenced event to the linking event using this relation. */
   reverseLinkRelation?: string;
-  /** Iff defined, this spec will create at most the given number of links to matching reference events.
+  /** If defined, this spec will create at most the given number of links to matching reference events.
```

Comment:
just as a note, the `iff`s here are "if and only if", not just typos. however in pretty much every case they're equivalent to the regular usage of "if" so idc if we change them.

Reply:
I'll leave them as is, they all seem like regular ifs the ones I found